mozdownload/cli.py

```diff
@@ -6,13 +6,15 @@
 
 """Tool to download different Gecko based applications (version {})."""
 
-from __future__ import absolute_import, print_function, unicode_literals
+from __future__ import absolute_import, unicode_literals
 
 import argparse
 import logging
 import os
 
-from mozdownload import __version__, factory, scraper
+from mozdownload import factory, scraper
+
+__version__ = '1.19'
```

Comment:
There is no good place in `__init__.py` for declaring this version without having pylama to blame about it. Given that only `cli.py` is using the variable beside `setup.py`, lets define it here.